server/views/pages/contacts/manage/contactDetails/language.njk

```diff
@@ -0,0 +1,28 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{{ govukSummaryList({
+    card: {
+        title: {
+            text: "Language details"
+        }
+    },
+    rows: [{
+        key: {
+            text: "Spoken language"
+        },
+        value: {
+            text: contact.languageDescription if contact.languageDescription else 'Not provided',
```

Comment:
can this be simplified to` text: contact.languageDescription || 'Not provided'`

Reply:
'or' operator added